Analysis/TrueJet/src/TrueJet.cc

```diff
@@ -837,6 +848,12 @@ void TrueJet::processEvent( LCEvent * event ) {
         // pre-PS part
 
       LCCollectionVec* fafpi_vec = new LCCollectionVec(LCIO::RECONSTRUCTEDPARTICLE ) ;
+      auto fafpiPidHandler = UTIL::PIDHandler(fafpi_vec);
+      auto fafpi_mainPidId = fafpiPidHandler.addAlgorithm("TrueJet_fafpi", {});
+      std::array<int, 25> fafpi_pidIds{};
```

Comment:
I know the 25 is just the result of correctly making the 0'th pid the mainPid, but I wonder if the 26 before was correct? I would have thought that we would expect an even number here (after removal of the mainPid).